### ISSUE TITLE
I couldn't figure this out

### ISSUE THREAD

**Issue by ghost** [learn-co-curriculum/node-debugger-lab#4]
I tried to ask a Learn Expert (no one apparently checks this track) and asked in Slack, but I couldn't get this built-in debugger working. I added 5 'debugger's in my code and ran `node debug account` and tried n, c, s and adding breakpoints, but nothing would stop the code from just jumping from line 1 to line 40...

**Comment by solomonp90** [comment]
Yeah I did literally the same exact thing and it didn't work for me either, not sure whats going on but it makes me wonder if i'm using debugger the right way
